### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by matheusfelipeog**
I did the preview application translation 👍

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -5,12 +5,14 @@
   "title": {
     "en_EN": "Preview",
     "nb_NO": "Forhåndsvisning",
-    "fr_FR": "Outil de prévisualisation"
+    "fr_FR": "Outil de prévisualisation",
+    "pt_BR": "Preview"
```
Comment (andersevenrud) [inline metadata.json]: And this one not translatable either ? :blush:

Comment (matheusfelipeog) [reply metadata.json]: That word yes, but we use "Preview" frequently in Brazil, just like "Music Player"

Comment (andersevenrud) [reply metadata.json]: Okay. Then you can go by the comment I left in the music player app repo :)

Comment (matheusfelipeog) [reply metadata.json]: So you can merge this too :)

**Pull request merged**